libp2p/identity/identify_push/identify_push.py

```diff
@@ -146,25 +147,26 @@ async def push_identify_to_peer(
         True if the push was successful, False otherwise.
 
     """
-    try:
-        # Create a new stream to the peer using the identify/push protocol
-        stream = await host.new_stream(peer_id, [ID_PUSH])
+    async with LIMIT:
```

Comment:
Can you make this in a way so that `LIMIT` uses the CONCURRENCY_LIMIT value as default but via a parameter you can specify the value on the fly